xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/XWikiContext.java

```diff
@@ -68,7 +69,7 @@
  *
  * @version $Id$
  */
-public class XWikiContext extends Hashtable<Object, Object>
+public class XWikiContext extends HashMap<Object, Object>
```

Comment:
XWikiContext being a very old public API and HashMap having different behavior than Hashtable regarding a few things this could break some use cases so I will close this Sonar issue as a false positive.